inc/zoo/swar/SWAR.h

```diff
@@ -160,29 +165,37 @@ constexpr auto isolate(T pattern) {
   return pattern & ((T(1)<<NBits)-1);
 }
 
+/// Clears the lowest bit set in type T
 template<typename T = uint64_t>
 constexpr auto clearLSB(T v) {
   return v & (v - 1);
 }
 
+// Sets only the loest bit set in type T
```

Comment:
Confusing comment, perhaps "leaves ON only the lowest bit set, or ~0 (all ones) for a zero input"

Reply:
It is confusing because it does not "set" a bit that was off (unless the input is zero)

Reply:
LOEST hahahaha